### DISCUSSION TITLE
Happy 8th Birthday VisiData!  🎂  ❤️

### DISCUSSION THREAD

**Discussion by frosencrantz** [saulpw/visidata#2589] [❤️]
Today is the 8th birthday for VisiData!

From the 5th birthday: https://www.visidata.org/blog/2021/5-years-old/

Your project keeps getting better, @saulpw!

**Comment by saulpw** [comment]
Thank you, @frosencrantz !!  The code keeps improving but also so does the community.  Good to have you (and everyone) here!
